src/dags/dag_update.py

```diff
@@ -26,18 +26,18 @@ def doit():
 
     fetch_dags_from_github = BashOperator(
         task_id="fetch_dags_from_github",
-        bash_command=f"wget {github_url} -O /tmp/repo.zip",
+        bash_command=f"wget {github_url} -O {SHARED_DIR}/repo.zip",
```

Comment:
I think this should be `${SHARED_DIR}`. Eg with a `$`

Reply:
it's an f string, not a bash command yet ;)

Reply:
Oh, no, not here, but you have also `BashOperators` where the string is not a format string. Than it boils down to shell variable expansion and then it does need the `$`